frontend/package.json

```diff
@@ -6,7 +6,8 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "dev": "next",
     "build": "next build",
-    "start": "next start"
+    "start": "next start",
+    "start:dev": "env WAMP_NEAR_EXPLORER_URL=wss://near-explorer-wamp.onrender.com/ws WAMP_NEAR_EXPLORER_FRONTEND_SECRET=Uitoo4Xe next"
```

Comment:
Well, `start:dev` is a confusing name. How do you feel about `dev:with-production-backend`? (It is the best non-confusing naming I came up with :disappointed: )

Reply:
A bit longer though 😓 
Ideal to have something smaller.